ambari-server/src/main/java/org/apache/ambari/server/view/ViewExtractor.java

```diff
@@ -64,11 +65,12 @@
    * @param viewArchive  the view archive file
    * @param archiveDir   the view archive directory
    *
+   * @param viewsAdditionalClasspath
```

Comment:
`@param` without description is invalid, please either add description or remove the tag.

Reply:
added description